fluentui_others/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.microsoft.fluentui" />
```

Comment:
same here, this is marked as fluentui but will be released as fluentui_others
but the fluentui.container will be relased as fluentui 
just wanted to know thoughts on this

Reply:
Refer to earlier comment : https://github.com/microsoft/fluentui-android/pull/41#discussion_r583870194